app/controllers/api/comments_controller.rb

```diff
@@ -0,0 +1,17 @@
+class Api::CommentsController < Api::ApplicationController
+  def index
+    @comments = Post.find(params[:post_id]).comments
+    render json: @comments
+  end
+
+  def create
+    @post = Post.find(params[:post_id])
+    @comment = Comment.create!(params.require(:comment).permit(:text)
+     .merge(author_id: params[:user_id], posts_id: @post.id))
```

Comment:
- [ ] I cannot create a comment. This should be `post_id`, not `posts_id`.

  ```
  .merge(..., post_id: @post.id))
  ```

  ![uiffk](https://user-images.githubusercontent.com/54848363/226100187-6c8ba938-3f71-44c0-953b-7d4c5f41b640.jpg)